src/hip_hcc.cpp

```diff
@@ -1604,7 +1610,7 @@ hipStream_t ihipPreLaunchKernel(hipStream_t stream, size_t grid, dim3 block, gri
 
 hipStream_t ihipPreLaunchKernel(hipStream_t stream, dim3 grid, size_t block, grid_launch_parm* lp,
                                 const char* kernelNameStr) {
-    HIP_INIT();
+    hip_impl::hip_init();
```

Comment:
Internal functions shouldn't call hip_init(), caller of this function should have already initialized the HIP.

Reply:
removed